multimedia/audacious-core/Portfile

```diff
@@ -73,21 +86,28 @@ variant gtk2 conflicts gtk3 description {Add GTK2 support} {
     depends_lib-append      path:lib/pkgconfig/gtk+-2.0.pc:gtk2
 
     configure.args-replace  -Dgtk=false \
-                            -Dgtk=true
+                            -Dgtk=true \
+                            -Dgtk2=false \
+                            -Dgtk2=true
 }
 
 variant gtk3 conflicts gtk2 description {Add GTK3 support} {
     depends_lib-append      path:lib/pkgconfig/gtk+-3.0.pc:gtk3
 
     configure.args-replace  -Dgtk=false \
                             -Dgtk=true
-
-    configure.args-append   -Dgtk3=true
 }
 
-# Need either one of gtk2, gtk3 or Qt5 to have a GUI.
-# On 10.7+ default to qt5, which is preferred by upstream.
+# Need either one of gtk2, gtk3, Qt5 or Qt6 to have a GUI.
+# Default to qt6, which is preferred by upstream.
```

Comment:
this comment doesn't seem to be true (here and in the other ports). As far as I can tell, you sill default to `qt5` on modern systems as the criteria below where you append `qt6` to default variants is only for non-darwin systems (also are you sure that appending to default variants is correct?

Reply:
@reneeotten AFAIU, GUI is supposed to be switchable, so Qt does not conflict GTK.

Reply:
okay, so that part might be okay then. It still doesn't seem to default to qt6 as advertised in the comment.

Reply:
@reneeotten That's only because that isn't currently working on macOS (I linked the issue). The intent is for that line to have `|| ${os.major} >= xx` where Qt 6 works. I'll look into it more later.

Regarding appending, I think @jmroot mentioned that not appending only works because of a bug.